app/assets/javascripts/topbar.js

```diff
@@ -9,17 +9,22 @@ $(function() {
 
 //open main menu on hover
 $(document).on('mouseenter', '.collapse .dropdown.menu_tab_dropdown', function(){
-  if(!$(this).hasClass('open')){
-    $(this).find('.dropdown-toggle:first').click();
+  var $this = $(this);
```

Comment:
refactor to use jquery best practice:
create cached jquery object $this which will be reused instead of repeating $(this)